.github/workflows/deploy_backend.yml

```diff
@@ -38,6 +39,10 @@ jobs:
         with:
           mask-password: true
 
+      - name: Copy AWS deployment files
+        run: |
+          cp --recursive deployment/aws/core_backend/* core_backend/
```

Comment:
We need to copy the scripts that Dockerfile.demo uses.

Reply:
Oh btw I changed the name from Dockerfile.demo to Dockerfile (in deployment/aws/core_backend) because the file doesn't seem to contain anything demo-environemnt-specific!

Reply:
Thanks!!